homeassistant/components/hassio/handler.py

```diff
@@ -118,6 +118,14 @@ def get_addon_info(self, addon):
         """
         return self.send_command(f"/addons/{addon}/info", method="get")
 
+    @api_data
+    def get_addons(self):
+        """Return data for all Add-ons.
+
+        This method return a coroutine.
+        """
+        return self.send_command("/addons", method="get")
```

Comment:
We should use the new endpoints: `/store/addons`

Reply:
just `/store` or `/store/repositories` in that case :+1: